roles/foreman_repositories/defaults/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 foreman_repositories_version: nightly
-foreman_repositories_use_release: True
-foreman_repositories_use_koji: False
+foreman_repositories_flavor: release
```

Comment:
Maybe comment the other option?

Reply:
You mean use_koji or describe what it is in a comment?